### DISCUSSION TITLE
Running Superset on Azure container Instance

### DISCUSSION THREAD

**Discussion by sunilk1303** [apache/superset#18661] [Q&A / Help]
Hi, 
I am trying to run superset on Azure container Instance with volume mapping on Azure file Share storage. When I build the container with below command the container instance gets on the running state but I can't launch the superset url. 

`az container create --resource-group $ACI_PERS_RESOURCE_GROUP --name superset01 --image superset-image:v1 --dns-name-label superset01 --ports 8088 --azure-file-volume-account-name $ACI_PERS_STORAGE_ACCOUNT_NAME --azure-file-volume-account-key $STORAGE_KEY --azure-file-volume-share-name $ACI_PERS_SHARE_NAME --azure-file-volume-mount-path "/app/superset_home/"`

Also I can see the volume map files are created on the file share but it doesn't grow to the initial size of superset.db.

```
l@Azure:~$ az storage file list -o table
Name                 Content Length    Type    Last Modified
-------------------  ----------------  ------  ---------------
cache/                                 dir
superset.db          0                 file
superset.db-journal  0                 file
```

Any inputs on this please?

**Comment by sfirke** [comment]
I spent a long time trying to run Superset with Docker on Azure Container Instances (and Azure App Services).  I was able to get the Superset container itself running fine but I wanted other containers in the group (Redis cache, Celery beat and worker) and there were too many obstacles to getting the group deployed.  Neither the Azure commands in the Docker CLI or the Azure CLI itself could quite get what I needed.

The only remaining route that had hope was using an [Azure-specific YAML file or Resource Manager template](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-vnet).  If I was trying to deploy many instances of Superset I would have explored this further but I'm just running a single instance.

I gave up and ran docker-compose on an Azure VM, with an Azure Flexible Postgres database back end.  It is a much simpler deployment route.  I'd hoped to avoid maintaining a VM but am happy with how it turned out.
